### DISCUSSION TITLE
New 2.4 Installation and no host alerts

### DISCUSSION THREAD

**Discussion by jmintha** [Security-Onion-Solutions/securityonion#11265] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

24

### RAM

192GB

### Storage for /

292GB

### Storage for /nsm

15TB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I have installed 2.4, everything seemed to go successfully.  I am getting various alerts from suricata for the network monitoring port.  I have added a host to the ElasticFleet and it enrolled and connected successfully and seems to be sending logs/etc.  However I am not seeing` any alerts in the Security Onion alert panel from the hosts.  The elastic-agent policy includes default integrations for: Elastic Defend, OSQuery Manager, System and Windows.

Previously I installed 2.3 and everything went fine, and I was getting a lot of alerts from the wazuh agents.  But with the elastic agent nothing.  (I've done a bunch of successful and unsuccessful sudos, altered important files and a few other things on the host which should generate some kind of alert.  

Did I miss some step?  Do I need to activate some rules?  If I look at the dashboard and include on the elastic agent, the "flow" type graph shows:
![elastic-agent](https://github.com/Security-Onion-Solutions/securityonion/assets/47866187/f156d722-b60e-4551-8dec-f1db88805d67)
The "*missing" seems important.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by ricamz** [comment]
I also have the same issue. Don't know if it is something that we have to configure.

**Reply by defensivedepth** [reply]
@ricamz Please start a new Discussion

**Comment by defensivedepth** [comment, marked as answer]
@jmintha Elastic Agent is setup by default on every grid node and ships local system logs to Security Onion. Detections can be enabled / built for these logs via Playbook. 

The screenshot you posted showing `*missing` just means that the field (for that viz, `event.category`) is missing from those logs.
